uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/request.controller.spec.js

```diff
@@ -593,261 +593,288 @@
       });
```

Comment:
Please revert changes and make two separate commits:
1) only wrap into a separate describe, nothing else
2) include new tests